beacon_node/beacon_chain/tests/validator_monitor.rs

```diff
@@ -234,21 +234,24 @@ async fn missed_blocks_basic() {
 
     // Let's fill the cache with the proposers for the current epoch
     // and push the duplicate_block_root to the block_roots vector
+    assert_eq!(
+        _state2.set_block_root(prev_slot, duplicate_block_root),
+        Ok(())
+    );
+
+    let decision_block_root = _state2
+        .proposer_shuffling_decision_root_at_epoch(epoch, Hash256::ZERO, &harness2.chain.spec)
+        .unwrap();
     assert_eq!(
         beacon_proposer_cache.lock().insert(
             epoch,
-            duplicate_block_root,
+            decision_block_root,
```

Comment:
Previously this test assumed that the `duplicate_block_root` was the decision root, but that is no longer true post Fulu.

Reply:
This one is wrong, yeah, but doesn't actually do anything, because the `block_root` argument to `proposer_shuffling_decision_root` isn't used unless we're right on an epoch boundary (pre-Fulu), or there's a whole epoch of skipped slots (post Fulu).

I think I would prefer to completely rewrite these tests than tweaking them any more. There is an open issue for this:

- https://github.com/sigp/lighthouse/issues/6293